### PR TITLE
Added command restrictions for AndroidSigningV2/V3, CopyFilesV2 tasks

### PR DESCRIPTION
**Pull request by anatolybolshakov**
**Task name**: AndroidSigningV2/V3, CopyFilesV2

**Description**: enabled command restrictions.

**Documentation changes required:** N

**Added unit tests:** N

**Attached related issue:** N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected - verified sanity scenarios for tasks, checked that setting of prohibited variables is not allowed for one of the tasks

### CODE REVIEW

File: Tasks/AndroidSigningV3/task.json [hunk 1]
```diff
@@ -198,5 +198,10 @@
         "CouldNotFindVersionOfToolInAndroidHome": "Could not find %s version of %s inside ANDROID_HOME: %s",
         "GetLatestToolVersion": "Trying to get the latest installed version of tool: %s",
         "GetSpecifiedToolVersion": "Trying to get version %s of %s"
+    },
+    "restrictions": {
+        "commands": {
+            "mode": "restricted"
+        }
```
Comment (mjroghelia) [inline Tasks/AndroidSigningV3/task.json:205]: Why no variable restrictions on this version?

Comment (anatolybolshakov) [reply Tasks/AndroidSigningV3/task.json:208]: Thanks 👍 Just missed them for this task - added also

**Review by EzzhevNikita — APPROVED** [review]
LGTM

**Review by mjroghelia — CHANGES_REQUESTED** [review]
@anatolybolshakov CMake shouldn't have these restrictions added. We can't assume any generic build tool (MSBuild, make, Gradle, Maven, Ant, etc) might not be legitimately outputting vso commands in one of the steps the build author wrote. See the reversion of the MSBuild task and subsequent Slack discussion. Feel free to ping me with questions.

**Comment by anatolybolshakov** [comment]
> @anatolybolshakov CMake shouldn't have these restrictions added. We can't assume any generic build tool (MSBuild, make, Gradle, Maven, Ant, etc) might not be legitimately outputting vso commands in one of the steps the build author wrote. See the reversion of the MSBuild task and subsequent Slack discussion. Feel free to ping me with questions.

@mjroghelia yep thanks, I removed these changes

**Review by alexander-smolyakov — APPROVED** [review]

**Review by mjroghelia — APPROVED** [review]

**Pull request merged**